src/main/java/com/redhat/coolstore/utils/DataBaseMigrationStartup.java

```diff
@@ -3,51 +3,44 @@
 import org.flywaydb.core.Flyway;
 import org.flywaydb.core.api.FlywayException;
 
-import javax.annotation.PostConstruct;
-import javax.annotation.Resource;
-import javax.ejb.Singleton;
-import javax.ejb.Startup;
-import javax.ejb.TransactionManagement;
-import javax.ejb.TransactionManagementType;
-import javax.inject.Inject;
+import jakarta.annotation.PostConstruct;
+import jakarta.annotation.Resource;
+import jakarta.inject.Inject;
+import io.quarkus.runtime.Startup;
+import io.quarkus.runtime.StartupEvent;
+
+import javax.enterprise.context.ApplicationScoped;
 import javax.sql.DataSource;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 /**
  * Created by tqvarnst on 2017-04-04.
  */
-@Singleton
+@ApplicationScoped
```

Comment:
Not sure why the `@Singleton` annotation has been removed, but the `@TransactionManagement` removal is correct. Below, the `@Resource` change should be swapped for `@Inject`, but I think we're missing a rule for that.